sci-geosciences/qmapshack/qmapshack-1.17.1_p20250426.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg
+
+COMMIT="948ae8a8a1526766ebdf9df2f207462c09c15c3b"
+
+DESCRIPTION="GPS mapping utility"
+HOMEPAGE="https://github.com/Maproom/qmapshack/wiki"
+SRC_URI="https://github.com/Maproom/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}"/${PN}-${COMMIT}
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="dbus"
+
+RDEPEND="
+	dev-db/sqlite
+	>=dev-libs/quazip-1.3:0=[qt6]
+	dev-qt/qt5compat:6
+	dev-qt/qtbase:6[dbus?,gui,network,sql,widgets,xml]
+	dev-qt/qtdeclarative:6
+	dev-qt/qttools:6[assistant,widgets]
+	dev-qt/qtwebengine:6[widgets]
+	sci-geosciences/routino
+	sci-libs/alglib
+	sci-libs/gdal:=
+	sci-libs/proj:=
+"
+DEPEND="${RDEPEND}"
+BDEPEND="dev-qt/qttools:6[linguist]"
+
+src_configure() {
+	local mycmakeargs=( -DUSE_QT6DBus=$(usex dbus) )
+	cmake_src_configure
+}
+
+src_install() {
+	docompress -x /usr/share/doc/${PF}/html
+	cmake_src_install
+	mv "${D}"/usr/share/doc/HTML "${D}"/usr/share/doc/${PF}/html || die "mv Qt help failed"
+}
+
+pkg_postinst() {
+	xdg_pkg_postinst
+
+	ewarn "Qt6 port: help system is currently broken"
```

Comment:
Is that still the case in git master?

Reply:
Yes, i will raise an issue upstream.

Reply:
https://github.com/Maproom/qmapshack/issues/728